src/app/dataset-view/additional-components/flows-component/components/flows-table/flows-table.component.ts

```diff
@@ -0,0 +1,107 @@
+    @Output() public filterByInitiatorChange = new EventEmitter<FilterByInitiatorEnum>();
+    @Output() public searchByAccountNameChange = new EventEmitter<string>();
+    public readonly DISPLAY_COLUMNS: string[] = ["description", "information", "creator", "options"];
+    public readonly INITIATORS: string[] = Object.keys(FilterByInitiatorEnum);
+    public readonly DEFAULT_AVATAR_URL = AppValues.DEFAULT_AVATAR_URL;
+    public readonly FlowStatus: typeof FlowStatus = FlowStatus;
+    public readonly FlowOutcome: typeof FlowOutcome = FlowOutcome;
+    public readonly FilterByInitiatorEnum: typeof FilterByInitiatorEnum = FilterByInitiatorEnum;
+
+    public dataSource: MatTableDataSource<FlowSummaryDataFragment>;
+    @ViewChildren(MatMenuTrigger) triggersMatMenu: QueryList<MatMenuTrigger>;
+
+    ngOnInit(): void {
+        this.dataSource = new MatTableDataSource(this.nodes);
+    }
+
+    public durationTask(d1: Scalars["DateTime"], d2: Scalars["DateTime"]): string {
+        const result = convertSecondsToHumanReadableFormat(moment(d2).seconds() - moment(d1).seconds());
+        return result ? result : "less than 1 second";
+    }
+
+    public descriptionForDatasetFlow(flow: FlowSummaryDataFragment): string {
+        return DataHelpers.descriptionForDatasetFlow(flow);
+    }
+
+    public descriptionColumnOptions(element: FlowSummaryDataFragment): { icon: string; class: string } {
+        return DatasetFlowTableHelpers.descriptionColumnTableOptions(element);
+    }
+
+    public descriptionDatasetFlowEndOfMessage(element: FlowSummaryDataFragment): string {
+        return DatasetFlowTableHelpers.descriptionEndOfMessage(element);
+    }
+
+    public descriptionDatasetFlowSubMessage(element: FlowSummaryDataFragment, fetchStep: FetchStep): string {
+        return DatasetFlowTableHelpers.descriptionSubMessage(element, fetchStep);
+    }
+
+    public changeFilterByStatus(status: MaybeNull<FlowStatus>): void {
+        this.filterByStatusChange.emit(status);
+    }
+
+    public onSearchByAccountName(): void {
+        this.searchByAccountNameChange.emit(this.searchByAccountName);
+        this.triggersMatMenu.get(1)?.closeMenu();
+    }
+
+    public changeFilterByInitiator(event: MatRadioChange): void {
+        this.filterByInitiatorChange.emit(event.value as FilterByInitiatorEnum);
+    }
+
+    public durationBlockVisible(node: FlowSummaryDataFragment): boolean {
+        return (
+            node.status === FlowStatus.Finished &&
+            ((node.description.__typename === "FlowDescriptionDatasetPollingIngest" &&
+                Boolean(node.description.ingestResult)) ||
+                (node.description.__typename === "FlowDescriptionDatasetPushIngest" &&
+                    Boolean(node.description.ingestResult)) ||
+                (node.description.__typename === "FlowDescriptionDatasetExecuteTransform" &&
+                    Boolean(node.description.transformResult)))
+        );
```

Comment:
I think it's enough that the flow is finished

Reply:
Fixed